### PR TITLE
testing CI using godot-tester

### PR DESCRIPTION
**Pull request by croconut**
Will update with v2.4 if standard is feasible. 
When running locally on standard linux godot version all tests passed.

### CODE REVIEW

File: src/core/MousePointer.gd [hunk 1]
```diff
@@ -89,9 +89,7 @@ func determine_global_mouse_pos() -> Vector2:
 	# scaling the game in full-creen.
 	if get_viewport() and get_viewport().has_node("Camera2D"):
 		zoom = get_viewport().get_node("Camera2D").zoom
-	var offset_mouse_position = \
-			get_tree().current_scene.get_global_mouse_position() \
-			- get_viewport_transform().origin
+	var offset_mouse_position = self.get_global_mouse_position() - get_viewport_transform().origin
```
Comment (croconut) [inline src/core/MousePointer.gd:92]: i think i may need to revert this.

Comment (db0) [reply src/core/MousePointer.gd]: so the .current_scene does exist in CLI?

**Comment by db0** [comment]
Thanks so much for helping me with this! I was fairly lost :)

Are you done with this? Can I merge?

**Comment by croconut** [comment]
Oh I wasn't done oh well. Pretty sure it runs faster using self instead of get_tree().current_scene and you'll want to update to v2.4 if standard is feasible in a docker container

**Comment by db0** [comment]
ok perfect. I'm working to get a linux VM to test and try to make it work on headless on my end. I'm waiting for 2.4 as well. Thanks again

**Comment by croconut** [comment]
I suspect it has to do with frame rate. The GitHub actions run on good computers and all but they might be lagging which might affect this since the yields aren't based on frame rate IIRC. When I locally test it passes even with headless, but I have a gaming rig. We could probably convert the testing yields to be in lock step with actual number of frames.

**Pull request merged**